### DISCUSSION TITLE
How to change a level's background

### DISCUSSION THREAD

**Discussion by zacnmac** [Techokami/SonicWorldsNext#111] [Q&A]
In Graphics/Background/, the actual background png is just this little picture of four squares, which looks good for base zone and all, but makes trying to make a level's background a forest or something of the sort pretty confusing. Any way to maybe make the background use a different image, or something similar to that?

**Comment by DimensionWarped** [comment]
This is less a Worlds question and more a basic Godot question. If you look at what is in the base zone/base zone act 2 right now, you'll see that there is no Worlds Next specific code that controls the parallax and instead it's just the built-in objects in Godot 4 (specifically the ParallaxBackground node and the ParallaxLayer node).

![image](https://github.com/Techokami/SonicWorldsNext/assets/7477617/9d925ed5-9e4e-45de-8d7b-d237c909dd61)

(note the script icon showing up in the HUD object above but not ParallaxBackground or any of its children)

That being the case, you can rest assured that you can just follow any Godot 4 (3 works too probably) parallax background tutorial on the Internet and expect the same results. No need to code your own like back in the MMF2 days.

I've taken the liberty of linking to you the favorite of the few I watched in the past ten minutes.
https://www.youtube.com/watch?v=f8z4x6R7OSM
